### ISSUE TITLE
[BUG] A11y - Landmarks should have a unique role or role/label/title (i.e. accessible name) combination

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/accessibility-mentor#36]
### Description

When running the a11y test
```
npx playwright test
```
following error occurs
```
Object {
+     "description": "Landmarks should have a unique role or role/label/title (i.e. accessible name) combination",
+     "help": "Ensures landmarks are unique",
+     "helpUrl": "https://dequeuniversity.com/rules/axe/4.8/landmark-unique?application=playwright",
+     "id": "landmark-unique",
+     "impact": "moderate",
+     "nodes": Array [
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "accessibleText": null,
+               "role": "navigation",
+             },
+             "id": "landmark-is-unique",
+             "impact": "moderate",
+             "message": "The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable",
+             "relatedNodes": Array [
+               Object {
+                 "html": "<div role=\"navigation\">",
+                 "target": Array [
+                   "div[role=\"navigation\"]",
+                 ],
+               },
+             ],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable",
+         "html": "<nav><ul><li><a href=\"#\">Home</a></li><li><a href=\"#\">Courses</a></li></ul></nav>",
+         "impact": "moderate",
+         "none": Array [],
+         "target": Array [
+           "nav",
+         ],
+       },
+     ],
+     "tags": Array [
+       "cat.semantics",
+       "best-practice",
+     ],
+   },
```

### Acceptance requirement

- [ ] update landmarks to dismiss the error

**Comment by Chizycodes** [comment]
Can you assign this to me? @YuriDevAT

**Comment by YuriDevAT** [comment]
Happy to do so, @Chizycodes 🔥
